dataframe/dataframe.go

```diff
@@ -1864,3 +1864,36 @@ type Matrix interface {
 	Dims() (r, c int)
 	At(i, j int) float64
 }
+
+// Describe prints the summary statistics for each column of the dataframe
+func (df DataFrame) Describe() DataFrame {
+	labels := series.Strings([]string{
+		"mean",
+		"stddev",
+		"min",
+		"25%",
+		"50%",
+		"75%",
+		"max",
+	})
+	labels.Name = "column"
+
+	ss := []series.Series{labels}
+
+	for _, col := range df.columns {
+		newCol := series.Strings([]string{
```

Comment:
We are returning a String series for all of them, maybe it would make more sense to return Float series when appropriate. In any case for now you can keep it as is and we might make improvements in the future if we deem it worth it.